src/docchecks.jl

```diff
@@ -255,6 +265,17 @@ function linkcheck(node::MarkdownAST.Node, doc::Document; method::Symbol=:HEAD)
     return false
 end
 
+
+function linkcheck(node::MarkdownAST.Node, docs_node::Documenter.DocsNode, doc::Document)
+    success = true
+    for mdast in docs_node.mdasts
+        success &= !(linkcheck(mdast, doc) === false)
+        # `linkcheck` may return `true` / `false` / `nothing`
```

Comment:
Just personally, I'm used to having the explanation before the behaviour:

```suggestion
        # `linkcheck` may return `true` / `false` / `nothing`
        success &= !(linkcheck(mdast, doc) === false)
```